ExtendedRelationOntology.ttl

```diff
@@ -42,6 +44,32 @@ cco:code_license rdf:type owl:AnnotationProperty ;
                  cco:is_curated_in_ontology "http://www.ontologyrepository.com/CommonCoreOntologies/Mid/ExtendedRelationOntology"^^xsd:anyURI ;
                  rdfs:label "code license"@en .
 
+###  http://www.ontologyrepository.com/CommonCoreOntologies/is_disrupted_by
+cco:is_disrupted_by rdf:type owl:ObjectProperty ;
+                    dcterms:created "2022-12-30T21:32:27-05:00"^^xsd:dateTime ;
+                    owl:inverseOf cco:is_disrupted_by ;
```

Comment:
I believe this says is_disrupted_by is the inverse of itself.

Reply:
Fixed.